extensions/stats/config.proto

```diff
@@ -42,4 +42,7 @@ message PluginConfig {
   // not available from the controlplane. Disable the fallback if the host
   // header originates outsides the mesh, like at ingress.
   bool disable_host_header_fallback = 6;
+
```

Comment:
Do these replace the standard metrics or add to them?
I think we need to specify replacing the whole set, or removal of a dimension.

Simpler api is: bool replace_dimension_set 
Default is false. This means that this list will add/replace existing dimensions.
If we need to remove you have to specify the whole thing .

Alternatively we should specify a repeated string dimensions_to_remove.
@douglas-reid  @kyessenov wdyt

Reply:
I'm happy with an override semantics. E.g. if it coincides with a default metric, then ignore the default computation, and an option to erase default metrics (for perf reasons?)

There's also a question whether this should be done across all 4 metrics or per metric.